depscan/lib/utils.py

```diff
@@ -413,22 +413,29 @@ def export_pdf(
 
 
 def render_template_report(
-    jsonl_report_file,
+    vdr_file,
     summary,
     template_file,
     result_file,
 ):
     """
     Render the given json_report_file and summary dict using the template_file with Jinja
     """
-    with open(jsonl_report_file, "r", encoding="utf-8") as jsonl_file:
-        json_report = [json.loads(jline) for jline in jsonl_file.readlines()]
+    if vdr_file and os.path.isfile(vdr_file):
+        with open(vdr_file, "r", encoding="utf-8") as f:
+            bom = json.load(f)
+    else:
+        bom = None
     with open(template_file, "r", encoding="utf-8") as tmpl_file:
         template = tmpl_file.read()
     jinja_env = Environment(autoescape=False)
     jinja_tmpl = jinja_env.from_string(template)
     report_result = jinja_tmpl.render(
-        vulnerabilities=json_report,
+        metadata=bom['metadata'] if bom else None,
```

Comment:
@heubeck I suggest doing this using the dictionary get method (e.g. `bom.get('metadata', None)`) and check for bom separately (or load the data from bom_file). If you think using the bom_file as a fallback seems reasonable, you'd definitely want to do this as the regular bom file will not include the vulnerabilities key.

```python
    report_result = jinja_tmpl.render(
        metadata=bom.get('metadata', None),
        vulnerabilities=bom.get('vulnerabilities', None),
        components=bom.get('components', None),
``` 
... and so on